utils/installer/install.ps1

```diff
@@ -0,0 +1,312 @@
+$ErrorActionPreference = "Stop" # exit when command fails
```

Comment:
What do we do with https://github.com/LunarVim/LunarVim/pull/1261 ?

Reply:
I just wanted the installer here to test the branch. I'll port the changes back to #1261 and remove the script from here before merging it.